cip.go

```diff
@@ -89,10 +89,18 @@ func main() {
 		"read all images in a repository and print to stdout")
 	snapshotTag := ""
 	flag.StringVar(&snapshotTag, "snapshot-tag", snapshotTag, "only snapshot images with the given tag")
+	minimalSnapshotPtr := flag.Bool(
+		"minimal-snapshot",
```

Comment:
I was thinking about "better" names, but I actually like that this is a statement of intent rather than just being narrowly focused on manifest lists.  :+1: